### DISCUSSION TITLE
求助，使用群晖vmm安装ppgw，不管怎么弄都是这种界面，谢谢

### DISCUSSION THREAD

**Discussion by laozhudu** [kkkgo/PaoPaoGateWay#80] [问和答]
![FireShot](https://github.com/kkkgo/PaoPaoGateWay/assets/36330302/dcf216ed-3452-4330-9d29-d03b7b2d1199)

**Comment by laozhudu** [comment]
希望大佬有个群晖vmm 转载的教程，谢谢

**Comment by kkkgo** [comment]
你配置的yaml模式，但你没放yaml文件呀

**Reply by laozhudu** [reply]
设置的是订阅链接

**Reply by kkkgo** [reply]
我觉得你把ppgw.ini贴出来比较好。
